### PR TITLE
Move creating appeal table to a Laravel job

### LINKED ISSUE #141: Convert appeal table updater to a Laravel Job
For cross-wiki support, this should support configuring a differently named page for different wikis (maybe linked via a Wikidata item?)

### PR DESCRIPTION
**Pull request by supertassu**
closes #141

### CODE REVIEW

File: app/Console/Kernel.php [hunk 1]
```diff
@@ -24,11 +24,13 @@ class Kernel extends ConsoleKernel
...
     {
+        // Backups
         $schedule->command('backup:run --only-db')->daily()->at('23:00');
         $schedule->command('backup:clean')->daily()->at('01:00');
         $schedule->command('backup:monitor')->daily()->at('05:00');
-        // $schedule->command('inspire')
-        //          ->hourly();
+
+        // Wiki integration
+        $schedule->command('utrs-jobs:update-appeal-tables --wiki=enwiki')->everyThirtyMinutes();
```
Comment (supertassu) [inline app/Console/Kernel.php]: not sure if this is ok or should it be more often

**Review by dqwiki — CHANGES_REQUESTED** [review]
While I don't think I ever implemented cookie storage on the python task, we do need to be logging in as little as possible. The examples given for addwiki/mediawiki-api do not indicate that this is something that already exists.

https://github.com/addwiki/mediawiki-api#example-usage

See also:
https://phabricator.wikimedia.org/T256533
https://www.mediawiki.org/wiki/API:Login#Additional_notes
https://en.wikipedia.org/wiki/User_talk:AmandaNP#High_DeltaQuadBot_login_rate
Likely irrelevant python theory: https://github.com/mwclient/mwclient/issues/256

**Comment by dqwiki** [comment]
I just reread #250 I'm assuming that solves the concern and has been merged?

**Comment by supertassu** [comment]
> I just reread #250 I'm assuming that solves the concern and has been merged?

@dqwiki If you're speaking about re-using sessions, nope, but I'm currently working on that. #250 does improve the situation so that UTRS will only log in when it needs to actually write something, otherwise it just uses the API anonymously

**Comment by supertassu** [comment]
see #347

**Review by dqwiki — APPROVED** [review]
Looks good from a code perspective.

**Pull request merged**